include/vectorSIMD.h

```diff
@@ -970,6 +970,22 @@ namespace core
     inline vectorSIMDIntBase& vectorSIMDIntBase::operator|=(const vectorSIMDf &other) { _mm_store_si128((__m128i*)this,_mm_or_si128(getAsRegister(),_mm_castps_si128(other.getAsRegister()))); return *this;}
     inline vectorSIMDIntBase& vectorSIMDIntBase::operator^=(const vectorSIMDf &other) { _mm_store_si128((__m128i*)this,_mm_xor_si128(getAsRegister(),_mm_castps_si128(other.getAsRegister()))); return *this;}
 
+	inline core::vectorSIMDf unpackLo(const core::vectorSIMDf& v0, const core::vectorSIMDf& v1)
+	{
+		core::vectorSIMDf result;
+		*((__m128*)(result.pointer)) = _mm_unpacklo_ps(v0.getAsRegister(), v1.getAsRegister());

```

Comment:
that's invalid, you need to use `_mm_store_ps`

Reply:
I've changed that already, but it worked

Reply:
its works because your compiler substitutes in `_mm_storeu_ps` silently